templates/admin-page.php

```diff
@@ -13,32 +13,32 @@
 
 	<div class="plugin-check-content">
 
-		<?php if ( ! empty( $available_plugins ) ) { ?>
+		<?php if ( ! empty( $available_plugins ) ) : ?>
 
 			<form>
 				<h2>
-					<label class="title" for="plugin-check__plugins">
+					<label class="title" for="plugin-check__plugins-dropdown">
 						<?php esc_html_e( 'Check the Plugin', 'plugin-check' ); ?>
 					</label>
 				</h2>
 
-				<select id="plugin-check__plugins" name="plugin_check_plugins">
+				<select id="plugin-check__plugins-dropdown" name="plugin_check_plugins">
 					<option><?php esc_html_e( 'Select Plugin', 'plugin-check' ); ?></option>
-					<?php foreach ( $available_plugins as $plugin_basename => $available_plugin ) { ?>
+					<?php foreach ( $available_plugins as $plugin_basename => $available_plugin ) : ?>
 						<option value="<?php echo esc_attr( $plugin_basename ); ?>"<?php selected( $selected_plugin_basename, $plugin_basename ); ?>>
 							<?php echo esc_html( $available_plugin['Name'] ); ?>
 						</option>
-					<?php } ?>
+					<?php endforeach; ?>
```

Comment:
Please revert this, we should use proper curly braces instead of that old syntax, which even in templates is discouraged at this point.